backend/src/main/java/com/cs471/studentLoanSystem/sql/LoanRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.cs471.studentLoanSystem.sql;
+
+import com.cs471.studentLoanSystem.sql.descriptions.Loan;
+import org.springframework.data.repository.CrudRepository;
+
+public interface LoanRepository extends CrudRepository<Loan, Integer> {
```

Comment:
I imagine having a `findAllByBankId` will be useful when we eventually display all loans of provided by a certain bank.

Reply:
Added this, along with "findAllByLoanStatus", since filtering by applied will be useful for bank officers.